crates/oxc_linter/src/rules/eslint/getter_return.rs

```diff
@@ -55,16 +55,19 @@ declare_oxc_lint!(
 
 impl Rule for GetterReturn {
     fn run<'a>(&self, node: &AstNode<'a>, ctx: &LintContext<'a>) {
+        // control flow dependant
+        let Some(cfg) = ctx.semantic().cfg() else { return };
```

Comment:
This was intentional, I was thinking that we can make the linter to omit CFG generation and disable the rules depending on it. Or the other way around if we have no rule dependent on CFG enabled we can disable the CFG generation altogether.
But we can go either way since it has no difference at the moment. I had no objection against unwrap/panics here.

Reply:
> if we have no rule dependent on CFG enabled we can disable the CFG generation altogether.

That's a nice optimization. I think that way around is the better option, in which case it could be `unwrap` here.

I have a broader feeling that we need a convention for communicating in code when (1) something is actually optional vs (2) there's an invariant that means this branch *must* always go one way. When delving into parts of Oxc codebase which I'm new to, this ambiguity has left me scratching my head quite a few times, wondering "in what circumstances can this happen?", and it turns out the answer is "it can't".

`unwrap`/`unreachable!` is one way to communicate that, but it has the downside of bloating the ASM with panic code for the unreachable branch. So I don't have a good answer to this.

Anyway, it's a broader problem, so I think we can just leave it as is in this PR, since I was too late with my review.